geometry_msgs/PoseStamped target_pose
float64 gripper_angle
int8 TYPE_PICK=0
int8 TYPE_PLACE=1
int8 TYPE_PICKPLACE=2
int8 type
